### PR TITLE
Add Reolink siren documentation

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add Reolink siren documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/88217
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -53,6 +53,13 @@ Depending on the supported features of the camera number entities are added for:
 - Optical zoom control
 - Focus control
 
+## Siren entities
+
+If the camera supports a siren, a siren entity will be created.
+When using siren turn on, the siren will continue to sound until the siren turn off is called.
+In some camera models there is delay up to 5 seconds between the turn off command and the sound stopping.
+The siren turn on supports duration (no turn off needed) and volume options.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]: Some rewording suggestions:

```suggestion
If the camera supports a siren, a siren entity will be created.
When using the siren turn-on service, the siren will continue to sound until the siren turn-off service is called.

In some camera models, there is a delay of up to 5 seconds between the turn-off command and the sound stopping. The siren turn-on service supports setting a volume and a duration (no turn-off service call is needed in that case).
```

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -87,3 +94,4 @@ The following models are lacking the HTTP webserver API and can therfore not wor
 - The Reolink NVR only sends event-notifications if motion happens on the camera connected to the first (index "0") channel, therefore the binary sensors of all channels will only be updated when the first channel sees motion.
 - Reolink doorbell presses only generate ONVIF event notifications when the doorbell is directly connected to your network. The doorbell visitor binary sensor will not work when connecting the Reolink doorbell to an NVR.
 - Older firmware versions do not expose the necessary information the integration needs to function. Ensure the camera is updated to the [latest firmware](https://reolink.com/download-center/) prior to setting up the integration.
+- The siren turn off command does not work on the Reolink NVR, you need to power cycle the NVR/camera to stop the siren. Reolink is aware of this firmware bug and is working on a solution.
```
Comment (frenck) [inline source/_integrations/reolink.markdown]:
```suggestion
- The siren turn-off service does not work on the Reolink NVR, you need to power cycle the NVR/camera to stop the siren. Reolink is aware of this firmware bug and is working on a solution.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by starkillerOG** [comment]
@frenck thanks for the improvements!

**Review by frenck — APPROVED** [review]
Thanks, @starkillerOG 👍 

../Frenck

**Pull request merged**